packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -120,18 +120,36 @@ export class ConfigUtil {
    * @returns The username for the configured Org if it exists.
    */
   public static async getUsername(): Promise<string | undefined> {
-    const configAggregator = await ConfigAggregatorProvider.getInstance().getConfigAggregator();
-    const defaultUsernameOrAlias = configAggregator.getPropertyValue(
-      OrgConfigProperties.TARGET_ORG
-    );
+    const defaultUsernameOrAlias = await ConfigUtil.getDefaultUsernameOrAlias();
     if (!defaultUsernameOrAlias) {
       return;
     }
 
-    const info = await StateAggregator.getInstance();
-    const username = defaultUsernameOrAlias
-      ? info.aliases.getUsername(String(defaultUsernameOrAlias))
-      : undefined;
+    const username = await getUsernameFor(defaultUsernameOrAlias);
+    return username ? String(username) : undefined;
+  }
+
+  /**
+   * Get the username of the default dev hub for the project.
+   *
+   * @returns The username for the configured default dev hub
+   * Org if it exists.
+   */
+  public static async getDevHubUsername(): Promise<string | undefined> {
+    const defaultDevHubUsernameOrAlias = await ConfigUtil.getDefaultDevHubUsernameOrAlias();
+    if (!defaultDevHubUsernameOrAlias) {
+      return;
+    }
+
+    const username = await getUsernameFor(defaultDevHubUsernameOrAlias);
     return username ? String(username) : undefined;
   }
 }
+
+async function getUsernameFor(usernameOrAlias: string) {
```

Comment:
Can this not use https://github.com/forcedotcom/salesforcedx-vscode/blob/a9826a7035254f54f45a0817ee19a25ab0645a16/packages/salesforcedx-vscode-core/src/util/authInfo.ts#L91

Reply:
ohhh...it's in utils and that method is in core.  bummer. Thine one should work though: https://github.com/forcedotcom/salesforcedx-vscode/blob/a9826a7035254f54f45a0817ee19a25ab0645a16/packages/salesforcedx-utils-vscode/src/auth/authUtil.ts#L44

Reply:
Yeah, good call.  Updated!